doc/api/modules.md

```diff
@@ -428,6 +428,29 @@ These are mostly for historic reasons.
 *Note*: It is strongly encouraged to place dependencies in the local
 `node_modules` folder. These will be loaded faster, and more reliably.
 
+## require() optional parameters
+<!-- YAML
+added: REPLACEME
+-->
+
+<!-- type=misc -->
+
+require() supports an optional parameter than can be used to pass
+an object containing modifiers to the require operation.
+
+Currently, the only property supported is `dlopenFlags`, which
+can be used to pass flags to the `dlopen` call that loads
```

Comment:
This sentence sounds a bit odd to me. Perhaps reword/restructure the whole section to something like:

>Valid properties:
>
>* `dlopenFlags` {integer} dlopen(3) flags to be used when loading a Node.js addon. Example:
>
>    ```js
>    var dlopen = require('os').constants.dlopen;
>    var foo = require('./foo.node', { dlopenFlags: dlopen.RTLD_NOW | dlopen.RTLD_GLOBAL });
>    ```